.github/workflows/ci.yml

```diff
@@ -0,0 +1,106 @@
+name: CI
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - prod-*
+      - stable-*
```

Comment:
Do we need this to run on push?

Reply:
Right now we don't have any CI runs on the code _after_ it is merged into `main` or other branches. This would let us know the status of the code in those branches not just the PR branches.

Reply:
The other reason for this is integrity of the CI status badge. With a test run against the stable branches, the CI status badge would only indicate that the last PR run was successful, not that the code in the stables branches was tested successfully.